tests/BootstrapperTest.php

```diff
@@ -82,11 +84,11 @@
     expect(DB::table('users')->first()->name)->toBe('Foo');
 });
 
-test('cache data is separated', function () {
+test('cache data is separated', function (string $bootstrapper) {
+    CacheManager::$addTags = true;
```

Comment:
Do we reset this static property anywhere? Tests should do that, see what we do with other static properties

Reply:
Like this? https://github.com/archtechx/tenancy/pull/1014/commits/66669934f91556802d89ef27c0f667ad0308cda7

Reply:
Yeah, just to the default values of the class. Should be done in both `beforeEach()` (so that the state is cleaned up before running the test) and in `afterEach()` (so that the state is cleaned up before running _other_ tests).

Reply:
And make sure to do it in every test file where the static properties get changed in any way.

Reply:
Seems like a lot of tests need this treatment – I'd probably do that in a separate PR. What do you think?

Reply:
What do you mean by that?

Reply:
> make sure to do it in every test file where the static properties get changed in any way

There are more test files (unrelated to this PR) in which the static properties get changed, but we don't reset them properly. I thought that since these files aren't related to this PR's changes, I'd add the proper static property resetting in another PR.

Reply:
Can you give some examples? We generally don't need to reset everything, it's usually just things we change in tests. You can see that we reset static properties in tests where we change them.

Reply:
E.g. in BootstrapperTest

https://github.com/archtechx/tenancy/blob/master/tests/BootstrapperTest.php#L384
https://github.com/archtechx/tenancy/blob/master/tests/BootstrapperTest.php#L385
https://github.com/archtechx/tenancy/blob/master/tests/BootstrapperTest.php#L415

Reply:
I see, those are all new features. Should ideally be added there as well. Just a `beforeEach()` and `afterEach()` to set all static properties manipulated in that test file to their default values.

Reply:
https://github.com/archtechx/tenancy/pull/1014/commits/53b2181779baa8541400dd014043636e213e6259